src/components/EventObjectives.astro

```diff
@@ -0,0 +1,25 @@
+<div class="w-full">
+
+    <div class="grid md:grid-cols-3 md:grid-rows-2 grid-rows-4 mb-16">
+        <h1 class="font-bold col-span-full text-center md:text-6xl text-5xl my-auto">Objetivos do Evento</h1>
+        <div class="col-span-1 my-auto">
+            <svg xmlns="http://www.w3.org/2000/svg" class="w-1/3 m-auto" viewBox="0 0 24 24"><path fill="currentColor" d="M12 22q-2.075 0-3.9-.788t-3.175-2.137q-1.35-1.35-2.137-3.175T2 12q0-2.075.788-3.9t2.137-3.175q1.35-1.35 3.175-2.137T12 2q2.075 0 3.9.788t3.175 2.137q1.35 1.35 2.138 3.175T22 12q0 2.075-.788 3.9t-2.137 3.175q-1.35 1.35-3.175 2.138T12 22m0-2q3.35 0 5.675-2.325T20 12q0-3.35-2.325-5.675T12 4Q8.65 4 6.325 6.325T4 12q0 3.35 2.325 5.675T12 20m0-2q-2.5 0-4.25-1.75T6 12q0-2.5 1.75-4.25T12 6q2.5 0 4.25 1.75T18 12q0 2.5-1.75 4.25T12 18m0-2q1.65 0 2.825-1.175T16 12q0-1.65-1.175-2.825T12 8q-1.65 0-2.825 1.175T8 12q0 1.65 1.175 2.825T12 16m0-2q-.825 0-1.412-.587T10 12q0-.825.588-1.412T12 10q.825 0 1.413.588T14 12q0 .825-.587 1.413T12 14"/></svg>
```

Comment:
Search here for the icons: https://icones.js.org/collection/mdi instead of pasting here the svg